sql/schema/v1-olap.sql

```diff
@@ -373,6 +373,28 @@ BEGIN
 END;
 $$;
 
+-- Events tables
+CREATE TABLE v1_events_olap (
+    tenant_id UUID NOT NULL,
+    id UUID NOT NULL,
+    seen_at TIMESTAMPTZ NOT NULL,
+    key TEXT NOT NULL,
+    payload JSONB NOT NULL,
+    additional_metadata JSONB,
+
+    PRIMARY KEY (tenant_id, id, seen_at)
+) PARTITION BY RANGE(seen_at);
+
+CREATE TABLE v1_event_to_run_olap (
+    run_id BIGINT NOT NULL,
+    run_inserted_at TIMESTAMPTZ NOT NULL,
+    event_id UUID NOT NULL,
+    event_seen_at TIMESTAMPTZ NOT NULL,
+
+    PRIMARY KEY (event_id, event_seen_at, run_id, run_inserted_at)
+) PARTITION BY RANGE(event_seen_at);
+
+
```

Comment:
was considering adding indexes here for filtering by key (at least) - curious for thoughts there